.github/workflows/github-pages.yml

```
@@ -0,0 +1,69 @@
+# This is a basic workflow to help you get started with Actions
+name: Deploy Sveltekit site to Pages
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "20"
+          cache: npm}
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+        with:
+          static_site_generator: sveltekit
+      - name: Restore cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            .sveltekit
+          # Generate a new cache whenever packages or source files change.
+          key: ${{ runner.os }}-sveltekit-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('./src/**/*.*', './static/**/*.*) }}
+          # If source files changed but packages didn't, rebuild from a prior cache.
+          restore-keys: |
+            ${{ runner.os }}-sveltekit-${{ hashFiles('**/package-lock.json') }}-
```

Ik denk dat deze trailing `-` weg kan?

```suggestion
            ${{ runner.os }}-sveltekit-${{ hashFiles('**/package-lock.json') }}
```

---

Nee dit is de fallback cache van regel `46` waar na de `-` de hash van files uit `rsc` en `static` komen.